.circleci/index.html

```diff
@@ -15,7 +15,7 @@
           </button>
         </div>
         <a class="pf-c-page__header-brand-link" href="#">
-          <img class="pf-c-brand pf-m-sm" src="https://pf4.patternfly.org/assets/images/l_pf-reverse-164x11.png" alt="Patternfly Logo" />
+          <img class="pf-c-brand pf-m-sm" src="https://pf4.patternfly.org/assets/images/PF-Masthead-Logo.svg" alt="Patternfly Logo" />
```

Comment:
Curious how this image is being used? I don't see it in the live page source

Reply:
@dlabrecq This is how the new logo is used when I open this file in my browser: 
<img width="739" alt="Screen Shot 2019-11-08 at 12 32 28 PM" src="https://user-images.githubusercontent.com/32821331/68498040-f3771500-0223-11ea-826f-dd2a94ee1e1f.png">